### DISCUSSION TITLE
Problema integrazione daikin onecta

### DISCUSSION THREAD

**Discussion by Obelix1975** [indomus/forum#3135] [GENERALE - Discussioni generiche]
Ciao a tutti,
sono da poco entrato anche io nel mondo di home assistant e cerco sempre di trovare soluzioni ai problemi cercando info e guide, ma mi trovo bloccato su questo problema da qualche giorno.
Ho dovuto cambiare il mio vecchio toshiba con un nuovo daikin e speravo di poterlo integrare facilmente in home assistant. Purtroppo l'adattatore LAN della mia unità è del nuovo tipo per cui non ci sono API direttamente compatibili in locale. 
A parte modifiche hardware, stavo cercando di usare l'integrazione daikin onecta.
Ho seguito la spiegazione per filo e per segno e tutto procede bene fino al momento in cui cerco di aggiungere l'integrazione.
A quel punto su HA si apre una finestra che chiede di aprire una pagina web, ma la pagina che si apre riporta solo la grafica daikin e una scritta bianca "proxy".
Da qui non riesco a proseguire in alcun modo e non riesco a capire dove sia il problema.
Ho usato da istruzioni il redirect uri: https://my.home-assistant.io/redirect/oauth ma se provo ad aprirlo direttamente nel mio browser mi appare un popup "invalid parameters given". Ad intuito direi che c'è un problema di comunicazione/autorizzazione del mio HA verso il sito daikin.
Ho provato ad abilitare "my home assistant" aggiungendo queste righe al mio config.yaml in cui effettivamente non erano presenti
```
# Example configuration.yaml entry
my:
```
ma nulla è cambiato.
Suggerisce di provare con <HOME_ASSISTANT_URL>/auth/external/callbackcome URI di reindirizzamento, suggerendo due esempi interni in http, ma http non è valido come redirect uri per l'app developer.
Sono bloccato qui e non capisco in che direzione devo andare per capire come risolvere. 
Per sostenere il progetto HA ho preferito fare l'abbonamento a nabu casa per poter accedere da remoto al mio HA, ho fatto alcune prove usando il link nabu casa, ma la procedura va in errore prima, e invece di arrivare alla pagina dove mi blocco adesso si apre una pagina "qualcosa è andato storto" prima.

Grazie in anticipo a chiunque voglia aiutarmi e scusate qualunque mio errore nella creazione di questo topic.
("sono un esordiente totale" cit.)

**Comment by robyseve78** [comment]
Anche a me succede la stessa cosa...hai risolto poi?

**Reply by Obelix1975** [reply]
Ciao, sì ho risolto cambiando browser. Ho installato e usato opera per fare la procedura e tutto ha funzionato regolarmente. Deve essere un problema di qualche blocco da parte dei browser come chrome ed edge. Prova e vedi se risolvi anche tu.

**Reply by alenik88** [reply]
provato anche con opera ma niente, sono su mac

**Comment by SimoneMais** [comment]
Buongiorno anche ame quando vado per la messa in funzione mi manda sulla pagina Daikin dove dice  "qualcosa è andato storto" e non riesco a fare altro,potete essermi di aiuto ? Grazieeeee

**Reply by alenik88** [reply]
idem con patate..... provato con diversi browser, chrome, brave, opera, safari.....niente

**Comment by Obelix1975** [comment]
Ciao, è passato qualche mese da quando ho aperto il topic, e i ricordi iniziano ad essere vaghi.
Da quello che ricordo, io avevo un problema "specifico": mi si apriva una pagina con una scritta "proxy" e la soluzione era quella di cambiare browser.
Però se non ricordo male anche a me ogni tanto durante la procedura usciva una pagina con scritto "qualcosa è andato storto" e non credo che il cambio di browser fosse efficace.
Mi pare che in quel caso il problema fosse nell'autenticazione al sito daikin: se avete fatto le varie procedure forse vi siete registrati un account direttamente sul sito, ebbene credo che il problema sia quello, nel senso che l'errore di qualcosa andato storto deriva dal fatto che usando quelle credenziali di accesso non si ottenga l'autorizzazione al login.
Sempre se ricordo bene, a quel punto facevo l'accesso tramite account google, che nel mio caso era la mail usata per creare l'account daikin precedente, e a quel punto l'autenticazione andava a buon fine.
Non garantisco che funzioni, ma fate un tentativo.

**Comment by SimoneMais** [comment]
Grazie mille farò altre prove ma ho cercato di andare dietro alla procedura
facendo la registrazione sul sito https://developer.cloud.daikineurope.com
e con le solite credenziali fatto quello per entrare in App Onecta Daikin
ma il problema della dicitura qualcosa e andato storto continua ad esserci
, stasera proverò ad entrare con account google e ti aggiorno.
Grazieeee

Il giorno mer 22 gen 2025 alle ore 14:38 Obelix1975 <
***@***.***> ha scritto:
